File: test_milestone1/repos/pure-js/src/greeter.js
// Greeting utilities.

function greet(name) {
  var message = formatGreeting("Hello", name);
  console.log(message);
  return message;
}

const formatGreeting = (template, name) => {
  return template + ", " + name + "!";
};

const farewell = (name) => {
  var msg = "Goodbye, " + name + "!";
  console.log(msg);
  return msg;
};

File: test_milestone1/repos/pure-rust/src/greeter.rs
pub fn greet(name: &str) -> String {
    let greeting = format_greeting(name);
    greeting
}

fn format_greeting(name: &str) -> String {
    format!("Hello, {}!", name)
}

pub fn farewell(name: &str) -> String {
    format!("Goodbye, {}!", name)
}
